modules/ROOT/pages/getting-started/index.adoc

```diff
@@ -4,17 +4,14 @@
 
 
 This tutorial walks you through creating a new project with the Neo4j GraphQL Library.
-
-If you are not familiar with Neo4j and GraphQL, you can alternatively take the course https://graphacademy.neo4j.com/courses/graphql-basics/?ref=docs[Introduction to Neo4j & GraphQL] in GraphAcademy to learn the fundamentals, how to use the xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox] and the Neo4j GraphQL Library to create GraphQL APIs backed by a Neo4j graph database.
```

Comment:
@lidiazuin
"Oh this should be kept actually, it was a request from DevRel"

there are two links to the graph academy course (index and getting-started/index)
we can leave it in both sites if that was the request.
i noticed however, that if you do the graph academy course, it leaves you with type definitions that deviate from what you setup with the tutorial. then again, the tutorial doesn't nearly cover as much.
maybe we could extend the tutorial?